package.json

```diff
@@ -88,6 +88,7 @@
     "pretty-fast": "^0.2.3",
     "prop-types": "^15.6.0",
     "react": "16.4.1",
+    "react-aria-components": "https://github.com/juanca/react-aria-components.git",
```

Comment:
I'll publish a new version to npm tomorrow, and if we're good with this library I'll point this to that version.